nautobot/docs/configuration/required-settings.md

```diff
@@ -135,12 +133,62 @@ setup](https://github.com/Suor/django-cacheops#setup).
 
 ### Task Queuing
 
-Task queues are configured by defining the `RQ_QUEUES` setting. Nautobot's core functionality relies on four distinct queues and these represent the minimum required set of queues that must be defined. By default, these are identical. It is up to you to modify them for your environment and know that other use cases like specific plugins may require additional queues to be defined.
+Task queues are configured by defining them within the [`RQ_QUEUES`](#rq_queues) setting. 
+
+Nautobot's core functionality relies on several distinct queues and these represent the minimum required set of queues that must be defined. By default, these use identical connection settings as defined in [`CACHES`](#caches) (yes, that's confusing and we'll explain below).
+
+It is up to you to modify the task queues for your environment and know that other use cases such as utilizing specific plugins may require additional queues to be defined. In most cases the default settings will be suitable for production use.
+
+#### CACHES
+
+The [`django-redis`](https://github.com/jazzband/django-redis) Django plugin is used to provide the backend for Redis as a concurrent write lock for preventing race conditions when allocating IP address objects and to define centralized Redis connection settings that will be used by RQ. The `CACHES` setting is required to to simplify the configuration for defining queues. *It is not used for caching at this time.*
+
+!!! important
+    Nautobot does not utilize the built-in [Django cache framework](https://docs.djangoproject.com/en/stable/topics/cache/) to perform caching because Cacheops is being used instead as detailed just above. *Yes, we know this is confusing, which is why this is being called out explicitly!*
```

Comment:
```suggestion
    Nautobot does not utilize the built-in [Django cache framework](https://docs.djangoproject.com/en/stable/topics/cache/) (which also relies on the `CACHES` setting) to perform caching because Cacheops is being used instead as detailed just above. *Yes, we know this is confusing, which is why this is being called out explicitly!*
```